api/v1alpha3/conditions_consts.go

```diff
@@ -0,0 +1,131 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha3
+
+import clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha3"
+
+const (
+	// VpcReady condition reports on the successful reconciliation of a VPC
+	VpcReadyCondition clusterv1.ConditionType = "VpcReady"
+	// VpcCreationStartedReason used when attempting to create a VPC for a managed cluster.
+	// Will not be applied to unmanaged clusters.
+	VpcCreationStartedReason = "VpcCreationStarted"
+	// VpcReconciliationFailedReason used when errors occur during VPC reconciliation
+	VpcReconciliationFailedReason = "VpcReconciliationFailed"
+)
+
+const (
+	// SubnetReady condition reports on the successful reconciliation of subnets.
+	SubnetReadyCondition clusterv1.ConditionType = "SubnetReady"
```

Comment:
Likely want to pluralize this one, since it would have to represent the aggregate of all subnets we are managing.
```suggestion
	SubnetsReadyCondition clusterv1.ConditionType = "SubnetsReady"
```